server/middlewares/uploadImage.js

```diff
@@ -0,0 +1,28 @@
+/* eslint prefer-const:0 */
+import cloudUpload from '../utilities/cloudUpload';
+/**
+ * This middleware handles the images upload to cloudinary
+ * @param {object} request the request from another middleware
+ * @param {object} response the response
+ * @param {object} next a callback function
+ * @returns {object} next() callback function
+ */
+const uploadImage = async (request, response, next) => {
+  const { image, images } = request.body;
+  let store = [];
+
+  if (image) {
+    await cloudUpload(store, image);
+    request.body.images = store;
+    return next();
+  }
+  // For an array of images.
+  if (images) {
+    await images.map(oneImage => cloudUpload(store, oneImage));
```

Comment:
`map` returns an array. If you're not using the return value it's cleaner to use a forEach

Reply:
Thanks